source/index.md

```diff
@@ -122,69 +125,70 @@ The table below [^2] summarizes the comparison of each standard:
+
+That means that the course, LMS and student progress were inseparable.
 
-xAPI specification removes content for it description, and allows the content to send “statements” based around [actor] [verb] [object], or “I – did – this” to a Learning Record Store (LRS) which can be part of Learning Management System but can live on their own or as part of another system. This was the first step for **Separation of concerns** in e-learning.
+xAPI specification removes content for it description, and allows the content to send “statements” based around [actor] [verb] [object], or “I – did – this” to a Learning Record Store (LRS) which can be a part of a Learning Management System, but can also live on its own (or as a part of another, separate, system). This was the first step for **Separation of concerns** in e-learning.
 
-### Learning Record Store LRS
+### Learning Record Store (LRS)
 
-A Learning Record Store is an external to course application that receives and sends data in JSON format from and to course runtime - it is an essential component in Experience API process flow. What's a big difference is that the specification does not tell how does course is being played (course runtime), it just defines that runtime does communicate with the interface (LRS) though xAPI Statements. The statements are open to extend, each implementation can introduce their own statements.
+A Learning Record Store is an application, external to course runtime, that receives and sends data in JSON format from and to the course runtime - it is an essential component in Experience API process flow. What is a big difference from previous e-learing standards, is that the specification does not tell how the course should be played (how course runtime should work), it just defines how the runtime must communicate with the interface (LRS) though xAPI Statements. The statements are open to be extended and each implementation can introduce its own statements.
 
 ![Experience API (xAPI) process flow with Learning Record Store (LRS)](images/lrs.png)
 
 ### cmi5 Specification
 
 > cmi5 is a "profile" for using the xAPI specification with traditional learning management (LMS) systems. [^6]
 
-The Specification of cmi5 is a set of rules providing all the capabilities of SCORM and xAPI.
-It is similar to SCORM in a way that it also contains XML file manifest, yet it does introduce the Assignable Unit (AU) - separately launchable learning content presentation. The AU is the unit of tracking and management. The AU collects data on the learner and sends it to the LMS.
+The specification of cmi5 is a set of rules providing all the capabilities of SCORM and xAPI.
+It is similar to SCORM in a way, as it also requires a XML file manifest, yet it does introduce the Assignable Unit (AU) - separately launchable learning content presentation. The AU is the unit of tracking and management. The AU collects data on the learner and sends it to the LMS (through an LRS).
```

Comment:
I'm not really sure of the whole `cmi5 Specification` section, you probably need to explain it to me in Polish first xd